Automattic/woocommerce-payments
# Remove Daily options from payout dropdown

Fixes https://github.com/Automattic/woocommerce-payments/issues/6895

#### Changes proposed in this Pull Request

For merchants in Japan, it's not possible to receive daily deposits. The default timing is 4 days.

This PR removes the "Daily" option from the deposits dropdown while leaving Weekly and Monthly.









#### Testing instructions





* Get this PR and run `npm start`
* Get a non-JP Stripe account and verify that the dropdown displays Daily, Weekly, and Monthly.
* Connect a JP account (ask for one in Slack if you don't have it)
* Make sure the Daily option isn't available

<img width="712" alt="Screenshot 2023-08-04 at 11 03 25" src="https://github.com/Automattic/woocommerce-payments/assets/12542046/d1c87389-e03a-4e06-a72d-9714922c1827">

-------------------


**Post merge**
